docs/MANUAL.md

```diff
@@ -326,11 +326,58 @@ Let op: als je KISS gebruikt in combinatie met een bronregister, waarin ook kana
 
 
-## Management informatie
-management informatie kan opgevraagd worden door rechtsreeks in de urlbalk van de browser naar
-[url van de website]/api/contactmomentendetails te gaan. Hij is alleen toegankelijk voor ingelogde gebruikers.
+## Managementinformatie
+Bij het opslaan van Contactmomenten worden enkele gegevens, die geen plek hebben in de standaarden rondom Klantinteracties, opgeslagen binnen KISS zelf. (zie ook het [onderdeel Contactmomentdetails in Decision Record](https://github.com/Klantinteractie-Servicesysteem/.github/blob/main/docs/DECISION-RECORD.md#contactmomentdetails)). Deze gegevens leveren managementinformatie over de werkzaamheden van het KCC. 
 
-Deze API haalt een lijst van contactmomentendetails op. Gebruik de onderstaande query parameters.
+Binnen KISS is een API endpoint beschikbaar waarmee deze gegevens opgevraagd kunnen worden. Dit endpoint is benaderbaar op: `https://www.kissbijdegemeente.nl/api/contactmomentendetails`.
+
+### Authenticatie
+
+De ContactmomentenDetails API is beveiligd en vereist een JWT Bearer Token voor toegang. Dit token wordt gegenereerd op basis van een geheime sleutel (secret) die door ons wordt verstrekt aan de externe systemen.
+
+#### Voorbeeld JWT Token
+
+Een voorbeeld van een JWT-token dat gebruikt kan worden voor authenticatie is:
+
+```
+eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoiRXh0ZXJuU3lzdGVlbSIsIm5iZiI6MTcyODU4MDUzMSwiZXhwIjoxNzI4NTg3NzMxLCJpYXQiOjE3Mjg1ODA1MzF9.QQNWDwsX9IeqCtqf_wFmJUW5iUJqGSPjYtOHWZG1RDc
+```
+
+#### Structuur van de JWT
+
+De JWT bestaat uit drie delen, gescheiden door punten:
+
+1. **Header**: Bevat informatie over het algoritme dat gebruikt wordt om het token te ondertekenen, in dit geval HS256.
+    ```json
+    {
+      "alg": "HS256",
+      "typ": "JWT"
+    }
+    ```
+
+2. **Payload**: De gegevens van de gebruiker of het systeem dat toegang probeert te verkrijgen. Dit bevat onder andere de rol `ExternSysteem`.
+    ```json
+    {
+      "role": "ExternSysteem",
+      "nbf": 1728580531,
+      "exp": 1728587731,
+      "iat": 1728580531
+    }
+    ```
+
+3. **Signature**: De handtekening, gegenereerd op basis van de header, payload, en een geheime sleutel (secret) die wij verstrekken. Dit zorgt ervoor dat het token niet kan worden gewijzigd door een derde partij.
+

```

Comment:
Misschien een hele blonde vraag: wat ik zie vande API's in bv. Open zaak of Open Klant, kun je het secret ergens instellen. 

Is dat hier ook zo? Waar kan je het secret vinden?

Reply:
@githubjimmeicatt , ik heb 
1. de nieuw EnvVar toegevoegd aan installatie.md
2. de envvar genoemd in manual.md
3. ik heb ook nog wat schrijfwijzes aangepast in het  stukje over de header, wil je dat nog controleren?

Reply:
> Misschien een hele blonde vraag: wat ik zie vande API's in bv. Open zaak of Open Klant, kun je het secret ergens instellen.
> 
> Is dat hier ook zo? Waar kan je het secret vinden?

Dit wordt een env variable. En hebben we besproken.



> @githubjimmeicatt , ik heb
> 
> 1. de nieuw EnvVar toegevoegd aan installatie.md
> 2. de envvar genoemd in manual.md
> 3. ik heb ook nog wat schrijfwijzes aangepast in het  stukje over de header, wil je dat nog controleren?

Gecontroleerd.